projects/observability/src/shared/components/bar-gauge/bar-gauge.component.ts

```diff
@@ -98,6 +98,9 @@ export class BarGaugeComponent implements OnChanges, AfterViewInit {
   @Input()
   public display: BarGaugeStyle = BarGaugeStyle.Regular;
 
+  @Input()
```

Comment:
nit: Since it only applies to single bar gauges right now we should include a comment here mentioning that

Reply:
Done